frontend/catalyst/pennylane_extensions.py

```diff
@@ -1239,6 +1247,17 @@ def decomposition(self):
         )
         return new_tape.operations
 
+    @property
+    def wires(self):
+        """The list of all control-wires, work-wires, and active-wires."""
+
+        assert len(self.regions) == 1, "Qctrl is expected to have one region"
+        total_wires = sum(
+            (op.wires for op in self.regions[0].quantum_tape.operations),
+            self._control_wires + self._work_wires,
+        )
```

Comment:
In this scenario, are the control wires coming first?

Reply:
yep, it's similar to PL

https://github.com/PennyLaneAI/pennylane/blob/343a6436bec71242ca20b3b28e24d1c83a8a99ed/pennylane/ops/op_math/controlled.py#L420

but I just realize that I should have added work_wires to the end of the list :)

Reply:
Haha nice :D